src/components/tasks/card/index.tsx

```diff
@@ -38,7 +39,7 @@ const Card: FC<Props> = ({ content }) => {
   function isTaskOverdue() {
     const currentDate = new Date();
     const timeLeft = localEndsOn.valueOf() - currentDate.valueOf();
-    return status !== COMPLETED && timeLeft <= 0;
+    return status !== COMPLETED && status !== UNASSIGNED && timeLeft <= 0;
```

Comment:
This should be `OR / ||` right?
Status can't be COMPLETED and UNASSIGNED at the same time.

Reply:
![image](https://user-images.githubusercontent.com/86315672/130566204-906ac768-456b-41e0-9941-8d00f8fdaaa1.png)
This is the result when I am using `AND / &&` operator 

![image](https://user-images.githubusercontent.com/86315672/130566335-2c09d8f0-e7ec-4fd6-970e-25490fc23289.png)
And this ,when I use `OR / || `operator

Reply:
The function should return `true` if the task is overdue and `false` otherwise.
Check if the function works properly and try to fix the implementation if there is any problem with the same.

Doesn't matter even if the output is correct when we can't understand the implementation.